src/helpers/index.ts

```diff
@@ -42,9 +37,7 @@ export const getBase255Number = (color: string, alpha = false): number => {
     }
     if (HEX.test(color)) {
         if (color.length === 3) {
-            return alpha
-                ? parseInt(color + color.slice(-1)) / 255
-                : parseInt(color + color.slice(-1));
+            color += color.slice(-1);
```

Comment:
Here it's not just a reformat, now it also takes into account the MAX_DECIMALS value